site/app/templates/admin/admin_gradeable/AdminGradeablePeers.twig

```diff
@@ -1,25 +1,25 @@
-<div id="peer-control">
+<div id="peer-control" data-testid="peer-control">
     <h2>Options for Peer Matrix</h2>
     <div class="row">
         <div class="col-md-6 pl-md-0">
             <div>
                 Upload a csv file dictating grader-student pairs for peer grading. The csv file must contain a
                 labelled 'student' column and a 'grader' column:<hr>
-                <input class="custom-file-input" type="file" name="peer_graders_list" id="peer_graders_list">
+                <input class="custom-file-input" type="file" name="peer_graders_list" id="peer_graders_list" data-testid="upload-peer-graders-list">
             </div>
         </div>
         <div class="col-md-6 pr-md-0">
             <div>
                 Enter number of peers for one student to grade and randomly assign peer grading in class.
                 <hr>
-                <div><input type="radio" id="all_grade" name="all_grade" class= "ignore" value="All Grade All" checked> All Grade All</div>
+                <div><input type="radio" id="all_grade" name="all_grade" class= "ignore" value="All Grade All" checked> <label data-testid="all-grade">All Grade All</label></div>
                 <div>
                     <input type="radio" name="all_grade" class="ignore" value="Number of Students">
-                    <input type="number" min="1" id="number_to_peer_grade" placeholder="Number of Students">
+                    <input type="number" min="1" id="number_to_peer_grade" placeholder="Number of Students" data-testid="number-to-peer-grade">
                 </div>
-                <div class="mt-3"><input type="checkbox" class="ignore" name="restrict-to-registration" id="restrict-to-registration" value="Restrict to Registration Sections">Restrict to Registration Sections</div>
+                <div class="mt-3"><input type="checkbox" class="ignore" name="restrict-to-registration" id="restrict-to-registration" value="Restrict to Registration Sections"> <label data-testid="restrict-registration-sections">Restrict to Registration Sections</label></div>
```

Comment:
Instead of adding labels, does adding a testid to the surrounding div work for this the other labels?